status/versions.go

```diff
@@ -185,11 +185,11 @@ func prometheusVersion() (*ExternalServiceInfo, error) {
 
 func kubernetesVersion() (*ExternalServiceInfo, error) {
 	product := ExternalServiceInfo{}
-	config, err := kubernetes.ConfigClient()
+	k8sConfig, err := kubernetes.ConfigClient()
```

Comment:
ah! that was the reason

Reply:
I would prefer changing the new local variable name than having a local variable hiding an import alias :)